templates/deployment.yaml

```diff
@@ -233,3 +237,178 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+
+# Netbox Worker
+---
+apiVersion: apps/v1
```

Comment:
Could you please make the creation of the worker conditional? e.g. only install it if `.Values.worker.enabled` is `true`. The default should be to install it, but I can see some people wanting not to have it.

Reply:
done